examples/doctest/CMakeLists.txt

```diff
@@ -6,17 +6,8 @@ project(CPMExampleDoctest)
 
 include(../../cmake/CPM.cmake)
 
-CPMAddPackage(
-  NAME fibonacci
-  GITLAB_REPOSITORY TheLartians/Fibonacci
-  VERSION 2.0
-)
-
-CPMAddPackage(
-  NAME doctest
-  GITHUB_REPOSITORY onqtam/doctest
-  GIT_TAG 2.4.5
-)
+CPMAddPackage("gh:TheLartians/Fibonacci@2.0")
```

Comment:
note that this changes the name of the package from `fibonacci` to `Fibonacci` - capitalized

Reply:
Good point it, imo the repo name `Fibonacci` should have been the correct name from the start. Shows that the new single-arg syntax is less error prone thanks to avoided repetition! 👍